### PR TITLE
Get client version dynamically

### PR DESCRIPTION
**Pull request by toddboom**
- [x] CHANGELOG.md updated
- [x] Rebased/mergable
- [ ] Tests pass
- [x] Sign [CLA](https://influxdata.com/community/cla/) (if not already signed)

### CODE REVIEW

File: services/admin/assets/js/admin.js [hunk 1]
```diff
@@ -343,7 +343,18 @@ var pretty = function(val) {
...
 
-var getVersion = function () {
+var getClientVersion = function () {
+    var query = $.get(window.location.origin + "/");
+
+    query.fail(handleRequestError);
+
+    query.done(function (data, status, xhr) {
+        var version = xhr.getResponseHeader('X-InfluxDB-Version');
+        $('.influxdb-client-version').html('v'+version);
```
Comment (gunnaraasen) [inline services/admin/assets/js/admin.js]: `X-InfluxDB-Version` returns `unknown` when InfluxDB is built without a version, which causes this to display as `InfluxDB vunkown`.

**Comment by gunnaraasen** [comment]
Had one comment, otherwise +1.

**Comment by e-dard** [comment]
Needs a rebase, but LGTM :+1:

**Comment by rossmcdonald** [comment]
@toddboom What needs to be done here to push this through? It looks like the 0.10.2 Admin UI wasn't updated, so the sooner we can get this in the better.

**Comment by toddboom** [comment]
@rossmcdonald i think @gunnaraasen was going to take a look at closing this out. i'll sync up with him today.

**Comment by gunnaraasen** [comment]
@toddboom I changed things around a bit. InfluxDB server version is now set using the header from the `getDatabases()` request and the `v` will only be appended when the version is not `unknown`.

**Comment by toddboom** [comment]
@gunnaraasen this looks good to me - keeps the code pretty clean and gets the job done. thanks! :+1:

**Pull request merged**